pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -224,7 +223,7 @@ func (b *OFBridge) Connect(maxRetrySec int, connectionCh chan struct{}) error {
 	b.maxRetrySec = maxRetrySec
 	b.connected = make(chan bool)
 	errCh := make(chan error)
-	sockPath := fmt.Sprintf("%s/%s.mgmt", OVSRunDir, b.bridgeName)
+	sockPath := fmt.Sprintf("%s%s.mgmt", OVSRunDir, b.bridgeName)
```

Comment:
Is it possible the OVSRunDir is different from the default value on the host? For example, if we use cloudbase OVS, and the default installation path might be C:\ProgramData\cloudbase\xxx, or the OVS is installed by user and the installation path is not C:\openvswitch.

What about using some environment variable to read the path? And we could add document to require user to set this environment variable in advance.

Reply:
@wenyingd Hi Wenying, I add "ovsRunDir" param in config file per our discussion. Could you help review the update?